### PR TITLE
Fixes #33969 - UX Review 3 - Tooltips - Assorted UI tweaks

### PR DESCRIPTION
**Pull request by Andrewgdewar**
#### What are the changes introduced in this pull request?
This PR covers all changes other than Empty state/spacing issues on the UX review seen [here](https://docs.google.com/document/d/1kFMBetXieUuAMc-UPc1_h2Gm9aY7JkL3Nn8bQvvz9EI).
(Pages 11 -27 inclusive)

#### What are the testing steps for this pull request?
Follow along with the changes requested from pages 11 - 27 within the [UX review Doc](https://docs.google.com/document/d/1kFMBetXieUuAMc-UPc1_h2Gm9aY7JkL3Nn8bQvvz9EI)

### CODE REVIEW

**Comment by jturel** [comment]
I see this PR is modifying `EditableTextInput` and so am I in another pr!

I'm requesting that this PR not be merged until #9816 is since that one will be ported back to Katello 4.3 and maybe even earlier versions and I'd like it to not depend on the changes here when cherry-picking backwards. It implies this PR will need a rebase once mine goes in. Does that work for you @Andrewgdewar ?

**Comment by jturel** [comment]
@Andrewgdewar this is ready to go for a rebase! My PR added a bunch of tests for EditableTextInput so should feel secure that passing tests means it works

File: webpack/scenes/ContentViews/Details/Filters/CVPackageGroupFilterContent.js [hunk 1]
```diff
@@ -216,54 +216,54 @@ const CVPackageGroupFilterContent = ({
...
+                </SplitItem>
+                <SplitItem>
+                  <Dropdown
+                    toggle={<KebabToggle aria-label="bulk_actions" onToggle={toggleBulkAction} />}
+                    isOpen={bulkActionOpen}
+                    isPlain
+                    dropdownItems={[
+                      <DropdownItem aria-label="bulk_add" key="bulk_add" isDisabled={!hasNotAddedSelected} component="button" onClick={bulkAdd}>
+                        {__('Add')}
+                      </DropdownItem>,
```
Comment (sjha4) [inline webpack/scenes/ContentViews/Details/Filters/CVPackageGroupFilterContent.js:249]: Do we remove Add from kebab menu here as per the review?

File: webpack/scenes/ContentViews/Details/Filters/Rules/Package/AddEditPackageRuleModal.js [hunk 2]
```diff
@@ -62,7 +62,7 @@ const AddEditPackageRuleModal = ({ filterId, onClose, selectedFilterRuleData })
 
   const formVersionParams = () => {
     switch (versionComparator) {
-      case 'All Versions':
+      case 'All versions':
```
Comment (sjha4) [inline webpack/scenes/ContentViews/Details/Filters/Rules/Package/AddEditPackageRuleModal.js:65]: Don't know when this changed but the dropdown selector for these options doesn't display properly in the form. Probably cause modal is set to high zIndex to not hide between nav..
We should convert the Select and SelectOption to FormSelect and FormSelectOption..
```
          <FormSelect
            value={versionComparator}
            onChange={onSelect}
            id="version_comparator"
            name="version_comparator"
            aria-label="version_comparator"
          >
            {
              Object.values(VersionModifiers).map(item => (
                <FormSelectOption key={item} value={item} label={VersionModifiers[item]}>
                </FormSelectOption>))
            }
          </FormSelect>
```

What do you think?

Comment (Andrewgdewar) [reply webpack/scenes/ContentViews/Details/Filters/Rules/Package/AddEditPackageRuleModal.js:65]: I can do that!

File: webpack/scenes/ContentViews/Details/Filters/CVPackageGroupFilterContent.js (hunk 1 above)
Comment (Andrewgdewar) [reply webpack/scenes/ContentViews/Details/Filters/CVPackageGroupFilterContent.js:249]: Yeah, we probably should. 
IF you think of any other locations let me know

File: webpack/scenes/ContentViews/Details/Versions/ContentViewVersionContent.js [hunk 3]
```diff
@@ -20,6 +21,14 @@ const ContentViewVersionContent = ({ cvId, versionId, cvVersion }) => {
...
   } = cvVersion;
 
+  const noCounts =
+    !Number(debCount) && !Number(dockerManifestCount) && !Number(dockerTagCount) &&
+    !Number(fileCount) && !Number(moduleStreamCount) && !Number(ansibleCollectionCount);
+
+  if (noCounts) {
+    return <InactiveText text={__('No content')} />;
+  }
+
```
Comment (sjha4) [inline webpack/scenes/ContentViews/Details/Versions/ContentViewVersionContent.js:31]: This doesn't take into account the generic content displayed which happens later in the file. There's code further down to show N/A, that can be modified.

File: webpack/scenes/ContentViews/Details/Versions/ContentViewVersionErrata.js [hunk 4]
```diff
@@ -15,6 +15,10 @@ const ContentViewVersionErrata = ({ cvId, versionId, errataCounts }) => {
     total,
   } = errataCounts;
 
+  if (!Number(total)) {
+    return <InactiveText text={__('No applicable errata')} />;
+  }
+
```
Comment (sjha4) [inline webpack/scenes/ContentViews/Details/Versions/ContentViewVersionErrata.js:21]: !Number(0) returns true..Didn't know that..

Comment (Andrewgdewar) [reply webpack/scenes/ContentViews/Details/Versions/ContentViewVersionErrata.js:21]: derp

File: webpack/scenes/ContentViews/Details/Versions/ContentViewVersionContent.js (hunk 3 above)
Comment (Andrewgdewar) [reply webpack/scenes/ContentViews/Details/Versions/ContentViewVersionContent.js:31]: On it

**Review by sjha4 — APPROVED** [review]
Code looks good..Everything worked as expected in my testing..Good to go 👍🏽

**Pull request merged**